rabbitmq/definitions.json

```diff
@@ -1,333 +1,275 @@
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"events",
-            "vhost":"/",
-            "destination":"survey.launched",
-            "destination_type":"queue",
-            "routing_key":"event.response.authentication",
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"notify.enriched.fulfilment.exchange",
-            "vhost":"/",
-            "destination":"notify.enriched.fulfilment",
-            "destination_type":"queue",
-            "routing_key":"",
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"uac-qid-created-exchange",
-            "vhost":"/",
-            "destination":"case.uac-qid-created",
-            "destination_type":"queue",
-            "routing_key":"",
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"events",
-            "vhost":"/",
-            "destination":"case.field.update",
-            "destination_type":"queue",
-            "routing_key":"event.fieldcase.update",
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"events",
-            "vhost":"/",
-            "destination":"Fulfilment.listener",
-            "destination_type":"queue",
-            "routing_key":"event.fulfilment.request",
-            "arguments":{
-
-            }
-        },
-        {
-            "source":"delayedRedeliveryExchange",
-            "vhost":"/",
-            "destination":"delayedRedeliveryQueue",
-            "destination_type":"queue",
-            "routing_key":"",
-            "arguments":{
-
-            }
-        }
-    ],
-    "policies":[
-        {
-            "name":"mirror",
-            "pattern":".*",
-            "vhost":"/",
-            "definition":{
-                "ha-mode":"exactly",
-                "ha-params":2,
-                "ha-promote-on-failure":"when-synced",
-                "ha-sync-mode":"automatic",
-                "ha-sync-batch-size":5000
-            }
-        }
-    ],
-    "parameters":[
-        {
-            "value": {
-	        "max-connections": 1000
-	    },
-	    "vhost": "/",
-	    "component": "vhost-limits",
-	    "name": "limits"
-        }
-    ]
+  "queues": [
+    {
+      "name": "events.caseProcessor.refusals",
+      "vhost": "/",
+      "durable": true,
+      "auto_delete": false,
+      "arguments": {
+        "x-dead-letter-exchange": "delayedRedeliveryExchange",
+        "x-dead-letter-routing-key": "events.caseProcessor.refusals"
+      }
+    },
+    {
+      "name": "events.caseProcessor.responses",
+      "vhost": "/",
+      "durable": true,
+      "auto_delete": false,
+      "arguments": {
+        "x-dead-letter-exchange": "delayedRedeliveryExchange",
+        "x-dead-letter-routing-key": "events.caseProcessor.responses"
+      }
+    },
+    {
+      "name": "supportTool.caseProcessor.sample",
```

Comment:
Maybe a flaw in this naming convention, do we think we'd rename this queue when we implement r-ops? At the moment both support tool and toolbox can publish to it. Highlights that direct queues don't always a have a single publisher.

Reply:
hmmm...  fair points,  no easy answer either.
One thing I've been wondering is: is there any plausible way to share definitions.json between all repos?  And for no hardcoding of queues/routing in any of the code.
ok this wouldn't fix the multiple publishers issues, but it would make renaming, adding, removing of queues far less painful.

Reply:
I guess it's more of an exception than the rule and it's still more useful having the 'main' publishing service in queue name.

It wouldn't certainly be nice to get away from copy pasting this sort of config everywhere, maybe one for a tech session?